RELEASE.md

```diff
@@ -3,7 +3,7 @@
 ## Overview 
 Release are created from the main repository branch using the version
```

Comment:
Should that read "Release is" or "Releases are"  ?

Reply:
Done.  Thanks.